packages/axelar-soroban-std/Cargo.toml

```
@@ -10,14 +10,15 @@ publish = true
 crate-type = ["rlib"]
 
 [dependencies]
+hex = { version = "0.4", optional = true }
```

```suggestion
hex = { workspace = true, optional = true }
```
use it from workspace everywhere since it's being shared across crates